html_css/v2/main-course/more_css/more-text-styles.md

```diff
@@ -0,0 +1,133 @@
+}
+~~~
+
+The point of this somewhat ridiculous string of font-families is try using the default font of the system's user interface. It will go through each of those fonts until it finds one that is installed on the system, and then use that. Using a stack like this often produces pleasing results, especially if you're going for a somewhat 'neutral' font style.
+
+#### Online Font Libraries
+One popular and easy method to get fonts that are _not_ installed on a user's computer is to use an online font library like [Google Fonts](https://fonts.google.com/), [Font Library](https://fontlibrary.org/) or the premium, but non-free [Adobe Fonts](https://fonts.adobe.com/).
+
+To use a font from one of these libraries, go to the website, select a font and then copy a snippet from the website to import that font from their server into your website. You'll be given either a `<link>` tag to put in your HTML like so....
+
+~~~html
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+~~~
+
+... or an `@import` tag that can be dropped at the top of a CSS file.
+
+~~~css
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+~~~
+
+Either method will import that font and make it available for you to use in your CSS:
+
+~~~css
+body {
+  font-family: 'Roboto', sans-serif;
+}
+~~~
+
+Keep in mind that it's important to add a fallback font.  If you're linking to an external API, you have no guarantee that the URL won't change, or that the external API won't go down at some point. Having a reasonable fallback means that if something goes wrong, at least your site won't look completely broken.
+
+#### Downloaded Fonts
+It is also possible to use a font that you have downloaded from the web. In your CSS file, you import and define a custom font using the `@font-face` rule, and then use as you would any other font-family.
+
+~~~css
+@font-face {
+  font-family: my-cool-font;
+  src: url(../fonts/the-font-file.woff);
+}
+
+h1 {
+  font-family: my-cool-font, sans-serif;
+}
+~~~
+
+This method _may_ be more reliable than relying on an third-party font API, but it is always wise to include a fallback.
+
+### Text Styles
+You learned the basics of manipulating fonts in our foundations lessons, but there is quite a bit more that you can do with CSS when it comes to manipulating text styles. These rules are all relatively simple and self-explanatory. You can refer to the docs for any questions you might have.
+
+#### font-style
+Typically used to make a font italic. You learned about the HTML `<em>` tag which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant, or should be emphasized in some way. There are cases where you may simply want text to be italic without having any special emphasis, and in those cases it is inappropriate to use `<em>`.
```

Comment:
Reworded the point while also making it more generic rather than focused strictly on the em tag. This point could also fit in the CSS Foundations lesson, depends whether we'd want people to have to be aware of it that early or not.

```suggestion
Typically used to make a font italic. You learned about the HTML `<em>` tag which uses an italic font, but `<em>` _also_ signifies that the text it wraps is significant, or should be emphasized in some way. A good rule of thumb to follow is that if you just want text to be visually/stylisticly italic (or bold, underlined, highlighted, etc.), use a CSS property. Otherwise if text should have some sort of semantic emphasis, use the correct HTML element.
```